clippy_lints/src/booleans.rs

```diff
@@ -44,6 +44,13 @@ declare_lint! {
     "boolean expressions that contain terminals which can be eliminated"
 }
 
+const METHODS_WITH_NEGATION: [(&str, &str); 4] = [
```

Comment:
What do you think about mentioning each pair only once and iterating over the list twice?

Reply:
Seems better indeed, changed.